base.php

```diff
@@ -713,7 +713,7 @@ function fixslashes($str) {
 	**/
 	function split($str,$noempty=TRUE) {
 		return array_map('trim',
-			preg_split('/[,;|]/',$str,0,$noempty?PREG_SPLIT_NO_EMPTY:0));
+			preg_split('/[,;|]/',$str??'',0,$noempty?PREG_SPLIT_NO_EMPTY:0));
```

Comment:
This change need PHP7 and break the requirements of php >= 5.4:
 https://fatfreeframework.com/3.7/system-requirements

Using the elvis operator (php5.3) will work without need a higher version.
`$str ?: ''`

Reply:
Good catch! :) Fixed it!

Reply:
Thank you!
One thing - the elvis operator is not null safe.
Maybe it's no matter in the context, but otherwise this will mitigate:
```
isset($str) ? $str : ''
```

Reply:
The ternary operator works with any truthy/falsy value. The `isset` is only needed when you want to e.g. fetch an array index that might not exist.